packages/y-indexeddb/src/index.ts

```diff
@@ -275,6 +281,9 @@ export const createIndexedDBProvider = (
       doc.off('destroy', handleDestroy);
     },
     async cleanup() {
+      if (connect) {
```

Comment:
I'm a bit confused. Is `connect` for connecting or connected?

Reply:
connected